backend/src/handlers/game.handler.ts

```diff
@@ -0,0 +1,74 @@
+import { Game, GameState, Player } from "../models";
+import { Server, Socket } from "socket.io";
+import { getGame } from "../services/game.service";
+import {
+  setMaxPlayers as setMaxPlayersService,
+  setRoundLimit as setRoundLimitService,
+} from "../services/game.service";
+
+export default (
+  io: Server,
+  socket: Socket
+): {
+  updateSetting: (
+    setting: "MAX_PLAYERS" | "ROUND_LIMIT",
+    value: number | undefined
+  ) => Promise<void>;
+} => {
+  const { gameCode } = socket.data;
+
+  const updateSetting = async (
+    setting: "MAX_PLAYERS" | "ROUND_LIMIT",
+    value: number | undefined
+  ): Promise<void> => {
+    if (isHost(socket, gameCode)) {
+      const game = await getGame(gameCode);
+      switch (setting) {
+        case "MAX_PLAYERS":
+          await setMaxPlayersService(game, value);
+          break;
+        case "ROUND_LIMIT":
+          await setRoundLimitService(game, value);
+          break;
+      }
```

Comment:
Is this only going to update one of those two settings at a time? If it isn't, then would it be better to if/else if for when both MAX_PLAYERS and ROUND_LIMIT are changed at the same time?

Reply:
This will only update one setting at a time as the user can only modify one at a time 👍🏼